app/views/how-tos/switching-from-govuk-prototype-kit.html

```diff
@@ -0,0 +1,105 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Switching from the GOV.UK prototype kit - NHS.UK prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "how-tos/includes/breadcrumb.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1>
```

Comment:
Can you add a class for the h1? Unlike govuk-frontend, more of the basic HTML tags do work without the classes, but for h1s especially we’re [encouraging people to uses the classes](https://service-manual.nhs.uk/design-system/styles/typography#headings), so that we can either drop the default or change it (from XL to L) in future.

```suggestion
      <h1 class="nhsuk-heading-xl">
```